### PR TITLE
Entry icons v2

### LINKED ISSUE #16: Entry icons
Support for custom entry icons

### PR DESCRIPTION
**Pull request by perry-mitchell**
image

Closes #16

### CODE REVIEW

File: src/main/menu.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ import { checkForUpdates } from './lib/updater';
 import { getMainWindow } from './utils/window';
 import i18n, { languages } from '../shared/i18n';
 import pkg from '../../package.json';
+import electronContextMenu from 'electron-context-menu';
+
+electronContextMenu();
```
Comment (perry-mitchell) [inline src/main/menu.js]: @sallar Is this still needed?

File: src/renderer/components/archive/copyable.js [hunk 2]
```diff
@@ -42,8 +43,8 @@ const Wrapper = styled(Flex)`
 const Content = styled.div`
   flex: 1;
   word-break: break-all;
-  padding: 8px 0 6px;
-  font-size: 0.95rem;
+  padding: 7px 0 6px;
+  /*font-size: 0.95rem;*/
```
Comment (perry-mitchell) [inline src/renderer/components/archive/copyable.js]: Can be removed..

File: src/main/menu.js (hunk 1 above)
Comment (sallar) [reply src/main/menu.js]: @perry-mitchell yeah, this was added by @hugo-cardenas and we discussed it in the initial PR. it adds some useful stuff -> https://github.com/buttercup/buttercup-desktop/pull/454#discussion_r155945312

File: src/renderer/components/archive/copyable.js (hunk 2 above)
Comment (sallar) [reply src/renderer/components/archive/copyable.js]: Good catch

**Comment by sallar** [comment]
@perry-mitchell done :)

**Comment by perry-mitchell** [comment]
Looks good to me @sallar!

**Pull request merged**